src/internal/utils/parseConfigsFromDocument.ts

```diff
@@ -0,0 +1,25 @@
+import { jsonFromEscapedJsonString } from "./jsonFromEscapedJsonString.ts";
+
+type documentConfigs = {
+  visualConfigurationData?: any;
+  themeData?: any;
+};
+
+export const parseConfigsFromDocument = (document: any): documentConfigs => {
+  if (document) {
+    const themeString =
+      document?._site?.pagesTheme?.[0]?.themeConfiguration?.data;
```

Comment:
Does this need to check that the site id on the theme and layout match the site id? Also need a way to handle dev mode but that's part of [SUMO-6628](https://yexttest.atlassian.net/jira/software/c/projects/SUMO/boards/97?selectedIssue=SUMO-6628)

Reply:
Fixed the issue, prod mode works as expected now.

Reply:
This works even if the site entity has multiple themes/layouts for different site ids?

Reply:
Updated to use resolveVisualEditorData and it does now